src/components/MeetYourHostLeft/MeetYourHostLeft.jsx

```diff
@@ -84,13 +88,45 @@ const PersonProfile = ({
           </div>
           <hr className={styles.borders} />
           <div className={styles.reviewsContainer}>
-            <span className={styles.reviewsNumbrs}>{yearsHosting}</span>
+            <span className={styles.reviewsNumbers}>{yearsHosting}</span>
             <span className={styles.reviewsText}>Year hosting</span>
           </div>
         </div>
       </div>
+
+      <div className={styles.personTextContainer}>
+        <div className={styles.personText}>
+          <div className={styles.languagesAndBusiness}>
+            {languagesAndBusinessData.map((item, index) => (
+              <div key={index} className={styles.lgAndBsText}>
+                <div className={styles.lgIcon}>
+                  <item.icon />
+                </div>
+                <div className={styles.lgText}>{item.text}</div>
+              </div>
+            ))}
+          </div>
+
+          <div className={styles.profileText}>{profileText}</div>
+        </div>
+
+        <div className={styles.arrowRightContainer}>
+          <span className={styles.showMore}>Show more</span>
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            viewBox="0 0 24 24"
+            width="24"
+            height="24"
+            fill="currentColor"
+            aria-label="arrow-right"
+            className={styles.arrowRight}
+          >
+            <path d="M9.29 15.88L13.17 12 9.29 8.12a.996.996 0 1 1 1.41-1.41l4.59 4.59a.996.996 0 0 1 0 1.41l-4.59 4.59a.996.996 0 1 1-1.41-1.41z" />
+          </svg>
```

Comment:
To make the code clean you can have this icon on the icon folder and import it from there.

Reply:
I also agree to change the location of icons to the icons folder.